test/cypress/utils/stripe-account-setup-stub.js

```diff
@@ -0,0 +1,69 @@
+const getGatewayAccountStripeSetupSuccess = function (opts) {
+  let stubOptions = {
+    gateway_account_id: opts.gatewayAccountId
+  }
+
+  if (opts.responsiblePerson !== undefined) {
+    stubOptions.responsible_person = opts.responsiblePerson
+  }
+  if (opts.bankAccount !== undefined) {
+    stubOptions.bank_account = opts.bankAccount
+  }
+  if (opts.vatNumber !== undefined) {
+    stubOptions.vat_number = opts.vatNumber
+  }
+  if (opts.companyNumber !== undefined) {
+    stubOptions.company_number = opts.companyNumber
+  }
+
+  return {
+    name: 'getGatewayAccountStripeSetupSuccess',
+    opts: stubOptions
+  }
+}
+
+const getGatewayAccountStripeSetupFlagForMultipleCalls = function (opts) {
```

Comment:
I think this stub is a bit overcomplicated, but I think we have a recommendation to replace the test that requires it, so should be able to get rid of it in not too long hopefully.